bigtable/client/cell.h

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_CELL_H_
+#define BIGTABLE_CLIENT_CELL_H_
+
+#include <absl/base/thread_annotations.h>
+#include <absl/strings/string_view.h>
+#include <bigtable/client/version.h>
```

Comment:
Please make that the first include, and change it to quotes:

```C++
#include "bigtable/client/version.h"
#include <absl/base/thread_annotations.h>
```

We have not documented the style guide in detail (WIP), sorry for the springing this rule on you.

Reply:
done